sdk/devcenter/developer-devcenter-rest/test/public/utils/recordedClient.ts

```diff
@@ -12,10 +12,18 @@ import { createTestCredential } from "@azure-tools/test-credential";
 
 const envSetupForPlayback: Record<string, string> = {
   ENDPOINT:
-    "https://8ab2df1c-ed88-4946-a8a9-e1bbb3e4d1fd-sdk-dc-na4b3zkj5hmeo.eastus.devcenter.azure.com/",
+    "https://8ab2df1c-ed88-4946-a8a9-e1bbb3e4d1fd-sdk-dc-4qb444onnk2tc.centraluseuap.devcenter.azure.com",
```

Comment:
Is this an actual endpoint?

`envSetupForPlayback` is meant for you to provide the fake value corresponding to an environment variable.
In this case, the actual endpoint will be replaced in the recordings with the fake value that you would provide here, and the recordings would allow you to run the tests in playback without actual secrets.

Though there is nothing related to keys/credentials that is being leaked here, it is generally a good practice to mask out your important env variables.